### DISCUSSION TITLE
Don't store kwargs in-memory

### DISCUSSION THREAD

**Discussion by WilliamDEdwards** [celery/celery#7478] [Q&A]
I disabled storing kwargs to my result backend by setting `result_extended` to false. However, it seems like Celery still stores these in-memory (`celery.events.state.Task`). Can this be disabled, i.e. have Celery pop all kwargs directly after the task has finished?

**Comment by WilliamDEdwards** [comment]
Should I open an issue for this to be added to the Celery 6.x roadmap?

**Reply by auvipy** [reply]
most probably something like this is still open in PR section

**Reply by WilliamDEdwards** [reply]
https://github.com/celery/celery/pulls?q=is%3Aopen+kwargs+ doesn't yield any appropriate results.
